db/migrate/20160808002834_add_files.rb

```diff
@@ -0,0 +1,29 @@
+class AddFiles < ActiveRecord::Migration
+  def change
+    create_table "katello_files" do |t|
+      t.timestamps
+      t.string "uuid", :null => false, :limit => 255
+      t.string 'name', :limit => 255
+      t.string 'checksum', :limit => 255
```

Comment:
I might would just have the full path and not the partial path.  That seems simpler and the user can do with it what they want.
```
name: "katello-ca-consumer-latest.noarch.rpm",
path: "pops/hey/WHAT/katello-ca-consumer-latest.noarch.rpm"
```